### DISCUSSION TITLE
【初めにお読みください】Qiita Discussions へようこそ

### DISCUSSION THREAD

**Discussion by kskwtnk** [increments/qiita-discussions#116] [アナウンス]
## 概要

質問や機能の要望などがありましたら、Qiita Discussions へご投稿ください。
Qiita Discussionsでは、ユーザーの皆様とのコミュニケーションを通じて、以下の項目を実現したいと考えています。

- より多くの方が意見を表明できる
    - 問い合わせに比べ、意見を発信しやすくなる
    - 既に挙がっている意見に対して共感を示せる
    - 運営からのメッセージに対してリアクションを示せる
- 意見が運営に届いているかが可視化される
    - 運営の対応予定の有無などを確認できる
- 障害情報を得やすくなる
    - 障害に対しての運営の対応状況や、復旧状況を確認できる

Qiita をより良いコミュニティにするために、皆様からの様々なご意見をお待ちしています！
（より詳細な背景はブログ記事 [GitHub Discussionsを正式に運用開始します](https://blog.qiita.com/qiita-discussions-general-availability/) をご覧ください。）

## 投稿方法
質問や機能の要望がありましたら、[Discussions](https://github.com/increments/qiita-discussions/discussions) から、New discussionを作成してください。作成にはGitHubアカウントが必要です。
![new_discussion](https://user-images.githubusercontent.com/103916185/173484191-3124353b-3072-4397-8bd8-43d4c3184950.png)

詳細は、公式のヘルプをご確認ください。
- [GitHub Docs](https://docs.github.com/ja/discussions/collaborating-with-your-community-using-discussions/participating-in-a-discussion)

## カテゴリについて
<img width="142" alt="category" src="https://user-images.githubusercontent.com/103916185/173484215-03327b27-8d0a-4720-b683-404705a8705a.png">

- アナウンス
    - 運営から新機能のリリースや、既存の機能の改善などをお知らせします
    - ユーザーの皆様からアナウンスカテゴリとしての投稿はできません
        - コメントは可能なため、更なる要望や不具合発見時はコメントいただけると幸いです
- ヘルプ
    - 運営チームやコミュニティへ、困ったことの共有
    - 「この内容、自分も困ったことがある！」と共感した場合は投票やコメントをお願いします
    - 運営チームからも回答をいたしますが、ユーザーの皆様からの回答もお待ちしています
- 要望・アイデア・不具合
    - Qiitaにあったら良いなと思う機能の要望やアイデア、または不具合の報告
        - こういう機能が欲しい、ここをこうやって改善したら使いやすくなるのではないか、そういった投稿をお待ちしています
        - 不具合かもしれない挙動を発見した際もこちらのカテゴリでの投稿をお願いします
    - 内容に共感した場合は投票やコメントをお願いします
- 障害情報
    - 障害が発生した際の状況共有
        - 障害の把握や対応状況について、こちらの場で共有します

## クローズ方法
クローズしたいDiscussionは、[クローズ機能](https://github.blog/changelog/2023-03-14-adding-the-ability-to-close-github-discussions/) を利用してクローズします。
運営およびDiscussionを作成した方ご自身でクローズが可能です。

## 以下の様な場合は、 [サポートフォーム ](https://support.qiita.com/hc/ja/requests/new)をご利用ください

- 特定のアカウントをご指摘いただく際
- パブリックな場では聞きづらい内容など

上記に該当するような内容ついては、削除させていただく可能性があります。
詳しくは下記「削除対応について」をご確認ください。

## 利用規約やコミュニティガイドライン違反の報告について

[利用規約](https://qiita.com/terms)や[コミュニティガイドライン](https://help.qiita.com/ja/articles/qiita-community-guideline)に違反している「記事」「コメント」「ユーザー」「Organization」を見つけた際は、[通報機能](https://help.qiita.com/ja/articles/qiita-report)をご利用ください。[コミュニティガイドライン](https://help.qiita.com/ja/articles/qiita-community-guideline#part-cf79556a40a47ce0)に沿って対応をしておりますので、個別に回答はいたしかねます。

## 削除対応について

当社または第三者の権利またはこれに類するものを害する恐れがあると当社が判断した投稿（セキュリティ、会社の技術・営業情報または個人情報保護の観点を想定していますがこれに限りません）は、権利保護の観点より該当する Discussion を削除させていただく場合があります。

なお、この場合、事前のご相談またはご連絡をしないまま削除する場合があることを予めご了承ください。

但し、誤った削除であることまたは、想定した恐れが生じないことが明らかであることを確認できた場合には、当該 Discussion を再掲する場合があります。

## その他

- [Code of conduct](https://github.com/increments/qiita-discussions/blob/main/CODE_OF_CONDUCT.md)
- [Security](https://github.com/increments/qiita-discussions/blob/main/SECURITY.md)

Qiita Discussions の活用方法として、ベータテスト時に行った以下のアナウンスも参考にしてみてください。
- [みなさまからいただいた意見をIssue化するフローについて · Discussion #19 · increments/qiita-discussions](https://github.com/increments/qiita-discussions/discussions/19)
- [現在Qiitaの開発で走っているプロジェクトの共有 · Discussion #33 · increments/qiita-discussions](https://github.com/increments/qiita-discussions/discussions/33)
- [Discussionをクローズするための運用を考えました · Discussion #49 · increments/qiita-discussions](https://github.com/increments/qiita-discussions/discussions/49)
- [「この記事はQiitaに投稿して良いのかな？」と迷ったら · Discussion #83 · increments/qiita-discussions](https://github.com/increments/qiita-discussions/discussions/83)

**Comment by my6129** [comment]
いつもお世話になります。
https://tech-diary.net/flask-introduction/保存版】30分でFlask入門！Webアプリの作り方をPythonエンジニアが解説を参考に勉強させて頂いております。
STEP5 : データベースの準備から前に進めません、from app import db     db.create_all()でエラーが出て　todo.dbが作成されずwebで探した内容を試しましたが解説及び動画通り進めませんなにかアドバイスがあればお願いいたします。

**Reply by yoshi389111** [reply]
通りすがりの者です（つまり、運営サイドの人ではありません）が、ここ Qiita Discussions は、Qiita  https://qiita.com/ の運営側と、利用者の間での意見交換などをする場です。

掲示のサイトは Qiita とは関係ないものだと思いますし、この Discussions は個々の技術の Q&A をする場でもありません。

質問などは、Qiita の質問 https://qiita.com/question-trend を使うか、その他の技術系の質問サイトで聞く方が回答をもらえる可能性があると思います。
必ず回答をもらえるとは限りませんが、少なくともここで質問するよりは回答がある可能性は高いと思います。

**Reply by tanaka-asami415** [reply]
@my6129 
yoshi389111 さんのコメントの通り
Qiita Discussionsは運営へのご質問やご意見を投稿していただく場となっておりますので
技術に関するご質問はQiitaの[質問機能](https://qiita.com/question-trend)をご利用ください🙇

@yoshi389111 
代わりにコメントいただきありがとうございます🙏

**Comment by EMUYN-LLC** [comment]
https://qiita.com/emuyn/items/cca82dce0f80a2a19f15
の記事の途中の部分ですが、
   $\bar{X}_{i}$ は第i条件の平均、
   $\bar{X}_{\text{Grand}}$ は全体の平均です。
の部分が数式で出力されません。
改行や表示位置を変えたときには正常にレンダリングされることもあるので、Qiita のシステムのバグではないかと思います。
ご対応いただけますと幸いです。
よろしくお願いします。
（再現のため、原稿はそのままにしてあります）

**Reply by tsubolabo** [reply]
@EMUYN-LLC 
　投稿先が間違っているようですが・・・。下記の一つ目の記述を二つ目か、三つ目のように変えると正常に表示されます。理由は分かりませんが、Qiita の数式ではときどきこのようなことがあります。早く改善して欲しいものです。

```
$\bar{X}_{i}$ は第i条件の平均、
   $\bar{X}_{\text{Grand}}$ は全体の平均です。

$\bar{X}\_{i}$ は第i条件の平均、
   $\bar{X}_{\text{Grand}}$ は全体の平均です。

$\bar{X}\_{i}$ は第i条件の平均、
   $\bar{X}\_{\text{Grand}}$ は全体の平均です。
```

**Reply by EMUYN-LLC** [reply]
ありがとうございました。助かりました。

**Reply by EMUYN-LLC** [reply]
"不具合かもしれない挙動を発見した際もこちらのカテゴリでの投稿をお願いします"
とあったのでここに Qiita system の不具合を書いたのですが、投稿先が間違っていたのですね。
申し訳なかったです。

**Reply by tanaka-asami415** [reply]
@EMUYN-LLC 
ご報告ありがとうございます。

ご不便をおかけして申し訳ありません。
数式内のアンダースコア '_' が複数ある場合に正しくパースされない場合があるのですが、解決にはお時間がかかってしまいそうです。

tsubolabo さんの記載のように、アンダースコア '_' の前にバックスラッシュ`\`をつけていただくか
`$\bar{X}_`と `{i}$ ` の間にスペースを入れていただくと回避していただけますので
改善まではこちらの方法で回避していただけると幸いです🙇

**Reply by P-SiZK** [reply]
@EMUYN-LLC 
大変お待たせしました :bow:

Markdown のエスケープを意識せずにインライン数式が記述できるように、インライン数式に新しい記法 ``$`...`$`` を導入しました！
[リリースノートはこちら](https://qiita.com/release-notes/2025-09-30)

今後とも Qiita をよろしくお願いいたします :pray:

**Comment by kt-1205** [comment]
https://qiita.com/minegishirei_v2/items/15aa6b4e67720bb75af1#comments
このサイトの画像
[イベント駆動アーキテクチャ]
をクリックしたら詐欺サイトにつながるPopupが出続ける悪質なアドウェアのサイトに飛びました。

**Reply by tanaka-asami415** [reply]
@kt-1205 
ご報告ありがとうございます。
リンク先の修正について、投稿者様に運営より[編集リクエスト](https://help.qiita.com/ja/articles/qiita-edit-request)を送信いたしました。

**Reply by kt-1205** [reply]
対応ありがとうございます

**Comment by yama-t** [comment]
記事のページで開発用のコンソールっぽいものが表示されっぱなしなんですが、
これはそういう感じの運用をされているのでしょうか？
![スクリーンショット 2024-12-13 11 55 22](https://github.com/user-attachments/assets/78294c74-de0b-483a-baec-953f4b4f107b)

**Reply by ohakutsu** [reply]
@yama-t 
ご報告いただきありがとうございます。
こちら確認させていただきます。

**Reply by kimkim0814** [reply]
@yama-t
お待たせしました！

記事のページで開発用のコンソールが表示される現象についてご報告いただきましたが、
現在この現象は収まっていることを確認いたしました。

調査の結果、Google Tag Manager に起因する現象の修正が行われた可能性が高いと考えられます。
Qiita側の問題ではないことが確認されており、現在は正常に動作していることを確認しております。ご安心ください。

**Comment by NewGoldSmith** [comment]
こんにちは
「この記事誰得？ 私しか得しないニッチな技術で記事投稿！」
このキャンペーンをまたやって頂けないでしょうか？

**Reply by kimkim0814** [reply]
@NewGoldSmith
キャンペーンへのリクエスト、ありがとうございます🙌
過去の企画を覚えていてくださって、「またやってほしい」と言っていただけるのは嬉しいです！

あのテーマだと、普段なかなか書けないニッチな内容も発信しやすくて良いですよね☺️
再開催について、社内で検討させていただきます。

**Comment by Koichi1981** [comment]
こんにちは。
Qiitaの有料会員から、解約、アカウント削除ができないのですが、どうすれば良いですか？
